sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -104,6 +104,13 @@ class SessionCatalog(externalCatalog: ExternalCatalog, conf: CatalystConf) {
     currentDb = db
   }
 
+  // todo: what is the default path in SessionCatalog?
+  def getDefaultPath: String = ""
+
+  def getDefaultDBExtension: String = ".db"
```

Comment:
either way I don't think we want this; just embed it in `getDefaultDBPath`. Otherwise we'll end up with too many random methods